morebits.js

```diff
@@ -2691,6 +2671,21 @@ Morebits.wiki.page = function(pageName, currentAction) {
 			// non-admin attempting to edit a protected page - this gives a friendlier message than the default
 			if (errorCode === 'protectedpage') {
 				ctx.statusElement.error('Failed to save edit: Page is protected');
+			// check for absuefilter hits: disallowed or warning
+			} else if (errorCode.indexOf('abusefilter') === 0) {
```

Comment:
Unnecessary busywork. Just put two separate `else if`s for `abusefilter-disallowed` and `abusefilter-warning`.

Reply:
Maybe it's just me, but I think cascading `if-else`s are generally easier to grok with relevant bits in one block, especially when they vary so much in size and depth, so readers can mentally say "okay this is the editconflict chunk, this is the blacklist chunk, this is the abusefilter chunk..."  And, at any rate, we'd have to duplicate the grabbing of `description` if separated.